paasta_tools/chronos_tools.py

```diff
@@ -377,11 +377,19 @@ def format_chronos_job_dict(self, docker_url, docker_volumes, dockerfile_locatio
         if not valid:
             raise InvalidChronosConfigError("\n".join(error_msgs))
 
+        net = self.get_net()
+        docker_chronos_net_mapping = {
```

Comment:
Please consolidate these mappings in mesos_utils, this is actually a mapper to the mesos protobuf:
https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L1689